docs/requirements.txt

```diff
@@ -17,10 +17,8 @@
 # under the License.
 #
 
-markdown==3.4.1
-recommonmark==0.7.1
-sphinx==4.5.0
-sphinx-book-theme==0.3.3
-sphinx-markdown-tables==0.0.17
+myst-parser==1.0.0
+sphinx==6.1.3
+sphinx-book-theme==1.0.1
```

Comment:
I have tried `sphinx-book-theme` 1.0.0 before, and I am not sure about the impacts from 1.x, as I didn't find the search bar on the left sidebar. It seems the search feature moved to header bar on the right corner.